src/main/java/uk/gov/pay/directdebit/tasks/resources/ExpireResource.java

```diff
@@ -27,7 +27,8 @@ public ExpireResource(ExpireService expireService) {
     @Produces(APPLICATION_JSON)
     public Response expirePaymentsAndMandates() {
         int numberOfExpiredPayments = expireService.expirePayments();
-        ResourceResponse resourceResponse =  new ResourceResponse(numberOfExpiredPayments);
+        int numberOfExpiredMandates = expireService.expireMandates();
```

Comment:
duplicate line

Reply:
Not sure I see the duplication, (wrt the ints) one is for payments and the other mandates? Sorry if I'm missing something obvious.

Reply:
oh yeah my bad